tidb-cloud/cli-reference.md

```diff
@@ -17,17 +17,19 @@ The following table lists the commands available for the TiDB Cloud CLI.
-| import     | cancel, describe, list, start                              | Manage [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
-| project    | list                                                       | Manage projects                                                                                          |
-| update     | -                                                          | Update the CLI to the latest version                                                                     |
+| Command            | Subcommand                                                               | Description                                                                                              |
+|--------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| serverless         | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
+| serverless branch  | create, delete, describe, list, shell                                    | Manage TiDB Serverless branches                                                                          |
+| serverless export  | create, describe, list, cancel, download                                 | Manage TiDB Serverless export jobs                                                                       |
+| serverless import  | cancel, describe, list, start                                            | Manage [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
+| auth               | login, logout                                                            | Login and logout                                                                                         |
+| ai                 | -                                                                        | Chat with TiDB Bot                                                                                       |
+| completion         | bash, fish, powershell, zsh                                              | Generate completion script for specified shell                                                           |
+| config             | create, delete, describe, edit, list, set, use                           | Configure user profiles                                                                                  |
+| help               | cluster, completion, config, help, import, project, update               | View help for any command                                                                                |
```

Comment:
Does it need to be listed in the TOC-tidb-cloud.md?

Reply:
I think `help` needn't to be listed